cmake/kokkos_arch.cmake

```diff
@@ -866,7 +868,7 @@ IF (KOKKOS_ARCH_VOLTA70 OR KOKKOS_ARCH_VOLTA72)
   SET(KOKKOS_ARCH_VOLTA ON)
 ENDIF()
 
-IF (KOKKOS_ARCH_AMPERE80 OR KOKKOS_ARCH_AMPERE86)
+IF (KOKKOS_ARCH_AMPERE80 OR KOKKOS_ARCH_AMPERE86 OR KOKKOS_ARCH_ADA89)
```

Comment:
That does not look right.
We do not define `KOKKOS_ENABLE_VOLTA` when using Turing75 and I fail to see why that is different.

Reply:
Ping

Reply:
You may be correct in principle, but my line of thinking is that there is no likely a need for specific optimizations for those consumer grade GPUs and go through the extra maintenance effort. Then grouping this with the closest feature set seems better than not grouping it into any of those at all. On the Kokkos slack it was confirmed by some Nvidia person that this assignment would be acceptable (I forgot who).

Bottom line, as far as I am concerned, you can take this, drop it, or modify it as you see fit. I personally don't see a need for a change and will not follow up any further. I've primarily submitted it since I consider it best practice to submit changes that we include in bundled software to the upstream version.